README.md

```diff
@@ -194,7 +232,15 @@ override fun onDestroy() {
 
 ---
 
-If you found potencial in this project please help us to improve the framework and help the open source community to have access to an awesome and well developed project!
+We are looking forward to improving this framework. If you have some feedback, comments, questions or if you want to contribute, join us at our [Slack group](https://join.slack.com/t/fluent-io/shared_invite/enQtMzQ3MDY3Njk0MjI3LTZkMjU0Y2Q3ZTA2NWQ0OTE4ZjFiZmQxNzJlMTYxNjIwODBjYmJhOGRlYzRhNGY5NmNjZTVhMTIzMTJhNzEwYTY)
+
+
+## Next Steps:
+* Library icon
+* Non-reactive samples
+* More reactive samples
+* Improve documentation with callbacks (non-reactive)
+* Fluent for iOS (Swift)
 
 ---
```

Comment:
I prefer to let the sample codes in documentation instead a link to the sample. Since the sample can change and we need to be aware to change the documentation too.